### ISSUE TITLE
Issues in 'Part 3 - Training Neural Networks (Exercises).ipynb'

### ISSUE THREAD

**Issue by HaleTom** [udacity/deep-learning-v2-pytorch#57]
## `Part 3 - Training Neural Networks (Exercises).ipynb`

Would you please put a random seed at the top of each notebook so that we can check our results against the `Solutions` .ipynb?

Eg:

    torch.manual_seed(1)
    if torch.cuda.is_available:
        torch.cuda.manual_seed_all(1)

The 'Part 1' notebook had this (but only for CPU). None of the following ones do.

---------------------------

In the section on `nn.LogSoftmax`, the exercise has the variable name `logits`, but the solution changes it to `logps` - it my be worthwhile updating the exercise to match.

---------------------------

In the looped network at the end, you have an `else:` after a for-loop with no `break`, which is unnecessary.

---------------------------

This notebook calls `model.forward(images)` twice.

This says that `.forward()` should be avoided as hooks don't run:
https://discuss.pytorch.org/t/any-different-between-model-input-and-model-forward-input/3690

---------------------------

In the final code block, `softmax()` is used when the instructions above say to use `exp()` if the final model operation is `LogSoftmax()`

---------------------------

I would make a PR for these myself, but am lacking the time.  Perhaps another motivated student wishes to do so.

**Comment by dJani97** [comment]
The manual seed would be a nice touch for most notebooks also because it would allow students to directly compare their results.

**Comment by abhiojha8** [comment]
We would put it for all notebooks when we plan a refresh. Thanks for the feedback :)
